pandas/tests/tools/test_datetime.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
```

Comment:
don't add a new file, instead add a test in pandas/tests/indexes/datetimes/test_tools.py

Reply:
Moved the test, thanks.